spec/integration/driver_spec.rb

```diff
@@ -1298,15 +1298,19 @@ def create_screenshot(file, *args)
       it "sends sequences with modifiers and symbols" do
         input = @session.find(:css, "#empty_input")
 
-        input.native.send_keys("t", "r", "i", "n", "g", %i[Ctrl Left], "s")
+        keys = Capybara::Cuprite.mac? ? %i[Alt Left] : %i[Ctrl Left]
```

Comment:
Hmm that's weird, phantomjs and poltergeist tests worked everywhere on all platforms. I wonder if it's Chrome issue or feature :)